README.md

```diff
@@ -210,7 +210,15 @@ If do not want to use the JavaScript API, but just the CLI you don't need Node.j
 
 ## Installation Instructions ##
 
-You can install the CLI using a pre-built binary or from source using Node.js.
+You can install the CLI from `npm`, using a pre-built binary or from source using Node.js.
+
+### Install from `npm` ###
+
+If you already have [Node.js](https://nodejs.org/en/download/) installed, you can install globally using `npm`:
+
+```sh
+npm install -g sfcc-ci
+```
```

Comment:
I think I have problems at times to launch the global binary using npm. It is clearly my stupid setup mixing nvm and brew, but maybe it'd be worth pointing out that one can always launch sfcc-ci using npx sfcc-ci

Reply:
@johnboxall Would you mind adding `npx sfcc-ci` as one alternative for using the CLI as @hnestmann suggested?

Reply:
Could you tell me more about why we'd recommend folks to use `npx` outside a specific set of one off circumstances (eg. you need to use exactly one command once from `sfcc-ci` in the context of a CI/CD pipeline)

It is significantly slower than installing it locally and using that:

```
$ node --version
v14.17.5
$ npm --version
6.14.14
$ time npx sfcc-ci
npx sfcc-ci  11.77s user 7.89s system 110% cpu 17.723 total
```

On my Mac, connected to blazing fast internet it takes ~12 seconds per invocation.

`npx` is great for tools like `react-create-app` or other things that you use once and never again. `sfcc-ci` doesn't feel like that to me.

Reply:
For me its quick if it's installed on the machine. I think the biggest benefit is that it always works, even if the global command couldnt be registered by npm

Reply:
Interesting! I see `npx` changed a changed a bunch in Node 16: https://docs.npmjs.com/cli/v7/commands/npx#compatibility-with-older-npx-versions

Using the version included in Node 16, I observe the caching behaviour you mention. 

Sure, I'll update the docs to mention that approach as well.